doc/lua_api.txt

```diff
@@ -3124,8 +3124,11 @@ These functions return the leftover itemstack.
 * `minetest.get_server_status()`: returns server status string
 * `minetest.get_server_uptime()`: returns the server uptime in seconds
 * `minetest.remove_player(name)`: remove player from database (if he is not connected).
-    * Does not remove player authentication data, minetest.player_exists will continue to return true.
+    * As auth data is not removed, minetest.player_exists will continue to return true.
+      Call the below method if you want to remove auth data too.
     * Returns a code (0: successful, 1: no such player, 2: player is connected)
+* `minetest.remove_player_auth(name)`: remove player authentication data
+    * Returns boolean indicating success
```

Comment:
should note that it should return false if the player doesn't exist

Reply:
i thought that was obvious since you can't <i>successfully</i> remove a nonexistant player
will add though

Reply:
I've seen similar (non-MT) APIs doing it either way, so it's good to mention